test/runnable/future.d

```diff
@@ -0,0 +1,43 @@
+/* PERMUTE_ARGS:
+ */
+
+class A
+{
+    @future char msg() { return 'a'; }
```

Comment:
I know, but I suspect having a hidden and unpredictable (for the user) list in the compiler is not going to go over very well. This sort of thing needs to be user-visible. It doesn't have to be named `@future` but it needs to be there.

Putting them in the compiler is more than just a name. It'd need to include the module, the class, the overloaded type, etc.